docs/pages/component_overview.js

```diff
@@ -0,0 +1,163 @@
+// @flow strict
+import { Fragment, useState, type Element, type Node } from 'react';
+import { Box, Fieldset, RadioButton, Flex, Text } from 'gestalt';
+import FOUNDATION_LIST from '../components/component_overview_foundations_list.js';
+import UTILITIES_LIST from '../components/component_overview_utilities_list.js';
+import BUILDING_BLOCKS_LIST from '../components/component_overview_building_blocks_list.js';
+import GENERAL_COMPONENT_LIST from '../components/component_overview_general_components_list.js';
+import Page from '../components/Page.js';
+import PageHeader from '../components/PageHeader.js';
+import IllustrationCard from '../components/IllustrationCard.js';
+import IllustrationSection from '../components/IllustrationSection.js';
+import IllustrationContainer from '../components/IllustrationContainer.js';
+import Accessibility from '../graphics/foundations/accessibility.svg';
+
+const getIllustrationCardColor = (category: string, isDark?: boolean) => {
```

Comment:
`isDark` isn't the most descriptive. It looks like this is toggling a dark background?

Reply:
+1, I'd update the name to hasDarkBackground here too

Reply:
DONE